bash_resources/healthcheck.sh

```diff
@@ -0,0 +1,53 @@
+function stop_fifachamps () {
+  kill $(lsof -c ng -t) > /dev/null 2>&1
+  kill $(lsof -c node -t) > /dev/null 2>&1 
+  kill $(pidof mongod) > /dev/null 2>&1 
+}
+
+function mongo_healthcheck() {
+  sleep 2
+  for i in `seq 1 5`; do
+    echo 'performing mongo healthcheck '${i}'/5'
+    if [ `ps -aux | grep mongod | wc -l` -ge '2' ]; then
+      echo 'mongo healthcheck passed'
+      return 0
+    fi
+    sleep 5
+  done
+
+  echo 'mongo healthcheck failed'
+  stop_fifachamps
+  exit 1
+}
+
+function api_healthcheck() {
+  for i in `seq 1 5`; do
+    echo 'performing api healthcheck '${i}'/5'
+    curl -sL -X GET localhost:5001/api/healthcheck > /dev/null
```

Comment:
I would like for URLs to not be hardcoded so we could configure them on different ports, I know that's not the way it is right now, so this is not an issue by itself, but on next interactions, try to think about that as well.